masonry_winit/src/event_loop_runner.rs

```diff
@@ -68,12 +68,19 @@ pub struct NewWindow {
 }
 
 impl NewWindow {
-    /// Create a new window with auto assigned [`WindowId::next()`].
+    /// A default `NewWindow`.
+    ///
+    /// Further properties of the window which will be created using
+    /// builder-style methods on this type.
+    /// See the documentation on the fields of this type for details of the parameters.
```

Comment:
Looking at this again, my use of the word "default" here wasn't great. Additionally, this API has changed since I wrote these docs. Perhaps something as simple as:

```suggestion
    /// Create a new window with an automatically assigned [`WindowId`].
    ///
    /// See the documentation on the fields of this type for details of the parameters.
```
is best.